src/ast/implementation/statement/revert_statement.ts

```diff
@@ -0,0 +1,29 @@
+import { ASTNode } from "../../ast_node";
+import { FunctionCall } from "../expression/function_call";
+import { Statement } from "./statement";
+
+export class RevertStatement extends Statement {
+    /**
```

Comment:
In 0.8.4 would `revert()` appear as the new `RevertStatement` node, or as an old-style `ExpressionStatement(FunctionCall...)`?

Reply:
As `RevertStatement`. It is a new standalone expression node like `EmitStatement`.